deployment/group_vars/dai.yml

```diff
@@ -39,3 +39,12 @@ UI_HOME_EXPLORER_ADDRESS_TEMPLATE: https://blockscout.com/poa/dai/address/%s
 UI_FOREIGN_EXPLORER_ADDRESS_TEMPLATE: https://blockscout.com/eth/mainnet/address/%s
 UI_HOME_GAS_PRICE_FALLBACK: 1000000000
 UI_FOREIGN_GAS_PRICE_FALLBACK: 10000000000
+
+## Monitor
+MONITOR_PORT: 3003
+MONITOR_HOME_DEPLOYMENT_BLOCK: 0
+MONITOR_FOREIGN_DEPLOYMENT_BLOCK: 0
+MONITOR_GAS_LIMIT: 300000
+MONITOR_GAS_PRICE_FALLBACK: 21
+MONITOR_LEFT_TX_THRESHOLD: 100
+MONITOR_CRON_SCHEDULE: "* * * * *"
```

Comment:
Currently we have separate schedules for checkWorker and ckeckWorker2: `0-59/3 * * * *` for the first one and  `1-59/3 * * * *` for the second one, so that at least for the first minute they don't overlap. This applies to all bridges.

Reply:
This would be done in #138 (Currently this schedule is not used)